app/controllers/katello/api/v2/content_view_filters_controller.rb

```diff
@@ -12,21 +12,28 @@ class Api::V2::ContentViewFiltersController < Api::V2::ApiController
     param_group :search, Api::V2::ApiController
     param :content_view_id, :identifier, :desc => N_("content view identifier"), :required => true
     param :name, String, :desc => N_("filter content view filters by name")
+    param :types, Array, :desc => N_("types of filters")
```

Comment:
This seems to be an existing issue unrelated to docker but the fix will be included in the PR